utils/unittest/irrecoverable.go

```diff
@@ -0,0 +1,31 @@
+package unittest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/onflow/flow-go/module/irrecoverable"
+)
+
+// IrrecoverableContextWithCancel returns an irrecoverable context, the cancel
+// function for the context, and the error channel for the context.
+func IrrecoverableContextWithCancel(ctx context.Context) (irrecoverable.SignalerContext, context.CancelFunc, <-chan error) {
```

Comment:
shall we move it to utils? Seems useful not only in tests

Reply:
https://github.com/onflow/flow-go/pull/2782/commits/a28465160f67de683aabcd94bd67411aaa6737a9 👍